source/connection-monitoring-and-pooling/tests/pool-clear-interrupt-in-use.yml

```diff
@@ -0,0 +1,44 @@
+version: 1
```

Comment:
I think this test can be removed now that we've added the MUST requirement of closing connections ASAP.

Reply:
done